### PR TITLE
expand image q formats and remove signature q

### PR DESCRIPTION
**Pull request by Robert-Costello**
## Technical Summary

as per discussion here https://github.com/dimagi/commcare-hq/pull/32026 allows .pdf,.doc, and .docx uploads in `ImageEntry` and removes (temporarily) `SignatureEntry`.
### QA Plan

Will go into QA when formplayer side is ready.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js [hunk 1]
```diff
@@ -975,21 +975,6 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
     VideoEntry.prototype = Object.create(FileEntry.prototype);
     VideoEntry.prototype.constructor = FileEntry;
 
-    /**
-     * Represents a signature, which requires the user to upload a signature file.
-     */
-    function SignatureEntry(question, options) {
```
Comment (snopoke) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js:981]: what happens if a form contains a signature question? Will it just not who up on the UI?

Comment (Robert-Costello) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js:981]: my understanding is that signature questions won't be an option initially and will be re-implemented later when drawing functionality is completed. 
@shubham1g5 @orangejenny please correct me if I'm missing something here.

Comment (orangejenny) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js:981]: yeah, that's the intent

Comment (shubham1g5) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js:981]: I think the behaviour on this should remain same as what's there on production today aka we just show a message "Sorry, web entry cannot support this type of question"

Comment (Robert-Costello) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js:981]: You get that message when the feature flag isn't on, correct? So for this change I should leave `SignatureEntry` as it is and just add the additional accepted file types to `ImageEntry`?

Comment (Robert-Costello) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js:981]: I see, thanks for the clarification! @orangejenny

**Review by snopoke — APPROVED** [review]

**Comment by snopoke** [comment]
Just to confirm, removing this widget isn't going to affect any existing apps right?

**Comment by Robert-Costello** [comment]
I checked and `WEB_APPS_UPLOAD_QUESTIONS` FF isn't enabled on any production domains. I also asked project teams on Slack, so it should be safe to merge.

**Comment by shubham1g5** [comment]
I am still seeing signature question enabled over [here](https://staging.commcarehq.org/a/shubhamgoyaltest/apps/view/54caa0c06cd2592c3189018601af2010/form/05dbe3c1e9d44019bc2d04d85369ab19/source/#form/sign). Any idea why?

**Pull request merged**